ci_framework/roles/cifmw_cephadm/README.md

```diff
@@ -58,6 +58,14 @@ need to be changed for a typical EDPM deployment.
 
 * `cifmw_cephadm_keys`: see below
 
+* `cifmw_cephadm_rgw_network`: the `storage_network` where the `radosgw` daemon
```

Comment:
I think these definitions (for rgw network and rgw vip) need to be updated because their values are now a function of the passed range and found IPs.

Reply:
Doing that in my next PS

Reply:
Done